processing/python/audience_view/surfaces.py

```diff
@@ -144,70 +145,70 @@ def calculate_circle_feed_positions(self):
             self.circle_feed_positions.append([lerp(self.start_x, self.goal_x, i/10.0), lerp(self.start_y, self.goal_y, i/10.0), lerp(0, 255, i/10.0)])
 
 
-class Alert(SurfaceBase):
-    circle_centers = {}
-    notify_info = {}
-    i = 0
-    a = 0
-    start_x = 0
-    start_y = 0
-    goal_x = 0
-    goal_y = 0
-    circle_feed_positions = []
-    col = color(0)
+# class Alert(SurfaceBase):
+#     circle_centers = {}
+#     notify_info = {}
+#     i = 0
+#     a = 0
+#     start_x = 0
+#     start_y = 0
+#     goal_x = 0
+#     goal_y = 0
+#     circle_feed_positions = []
+#     col = color(0)
```

Comment:
You might as well just delete the class, no?

Reply:
done